miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppInfo.kt

```diff
@@ -15,8 +15,8 @@ import kotlinx.android.parcel.Parcelize
 data class MiniAppInfo(
     // Must use @SerializedName on all properties for compatibility with Proguard obfuscation
     @SerializedName("id") val id: String,
-    @SerializedName("displayName") val displayName: String,
-    @SerializedName("icon") val icon: String,
+    @SerializedName("displayName") val displayName: String?,
```

Comment:
Does the BE API Contract also mention the nullability?

Reply:
No but there is ticket for it https://jira.rakuten-it.com/jira/browse/MINI-824

Reply:
Can we confirm with BE on the ticket or ask for clarity, from the ticket it looks that it is solved there with API changes?

Reply:
It's not null from the backend but the reason we have that ticket is that we don't want to rely on the nullability from the server 🤔
`displayName` and `icon` are not optional.